src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -14799,34 +14775,31 @@ class FindRelatedBooksAsyncFlattenedAsyncPagedAllOdyssey
    /// <summary>
    /// Testing calling forms
    /// </summary>
-   public static void SampleFindRelatedBooks()
+   public static async Task SampleFindRelatedBooksAsync()
    {
-       public static async Task SampleFindRelatedBooksAsync()
+       LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
+       IEnumerable<BookName> names = new[]
        {
-           LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-           IEnumerable<BookName> names = new[]
-           {
-               new BookName("[SHELF_ID]", "[BOOK_ID]"),
-           };
-           IEnumerable<ShelfName> shelves = new[]
-           {
-               new ShelfName("[SHELF_ID]"),
-           };
-           gax::PagedAsyncEnumerable<FindRelatedBooksResponse, BookName> response = libraryServiceClient.FindRelatedBooksAsync(names, shelves);
-           // Iterate over pages (of server-defined size), performing one RPC per page
-           foreach (BookName item in response)
-           {
-               // FIXME: inspect the results
-           }
-       }
-       // [END sample_core]
+           new BookName("[SHELF_ID]", "[BOOK_ID]"),
+       };
+       IEnumerable<ShelfName> shelves = new[]
+       {
+           new ShelfName("[SHELF_ID]"),
+       };
+       gax::PagedAsyncEnumerable<FindRelatedBooksResponse, BookName> response = await libraryServiceClient.FindRelatedBooksAsync(names, shelves);
+       // Iterate over pages (of server-defined size), performing one RPC per page
+       await response.ForEachAsync((BookName item) =>
+       {
+           // FIXME: inspect the results
+       });
+   }
+   // [END sample_core]
    }
 
    // [END sample]
-   public static void Main(String[] args)
+   public static Task Main(string[] args)
    {
-   	// FIXME: pass in commandline arguments
-   	// FIXME: call the sample function
+   	await SampleFindRelatedBooks();
```

Comment:
Just noticed that this should probably be calling `SampleFindRelatedBooksAsync()`.